### DISCUSSION TITLE
What are the code translation settings for Granite Code models?

### DISCUSSION THREAD

**Discussion by Lalit439cs** [ibm-granite/discussions#5] [Q&A] [answered]
While trying Code tranlation on Code Lingua benchmark, what is the prompt setting for results shown in the paper ? Is it default prompt type, mentioned in the code of Code Lingua benchmark?

**Comment by rpand002** [comment, marked as answer]
Yes, it is same as the one mentioned in Code Lingua benchmark.

**Reply by Lalit439cs** [reply]
Thanks for quick response!
